client/assets/stylesheets/style.scss

```diff
@@ -4,6 +4,9 @@
 // Color Schemes
 @import '~@automattic/calypso-color-schemes/src/calypso-color-schemes';
 
+// WordPress Components
+@import '~@wordpress/components/build-style/style';
```

Comment:
Note that core component styles are not scoped, so there could be CSS leaking in/out from them.

I don't know if there actually are clashing CSS selectors but it's possible.